dss/events/handlers/index.py

```diff
@@ -1,21 +1,25 @@
+
+
```

Comment:
Never mind. The line numbering on the diff render in GitHub is confused. Please just submit smaller diffs in the future.